lib/pbench/cli/agent/commands/tools/clear.py

```diff
@@ -122,6 +55,81 @@ def execute(self) -> int:
 
         return errors
 
+    def _clear_remotes(self, group) -> Union[int, list]:
```

Comment:
I think the return type should be `Tuple[int, list]` -- if you wanted the option to also return `None`, then the return type would be `Union[Tuple[int, list], NoneType]`, or something like that.

[I see that Dave noted this, too; and, his suggestion is better.  🙂]